### PR TITLE
max response

### PR DESCRIPTION
**Pull request by doremaxime**
Comfort: 4
Clarity: 4

I used class notes.
Wasn't too sure about the "Please do not use data = getFormFields(form)
instead write out the data object."

### CODE REVIEW

File: diagnostic.md [hunk 1]
```diff
@@ -21,13 +21,19 @@ Write the `curl` request you'd use to retrieve a list of **all donuts** on the
 server.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts \
+--header "Content-Type: application/json"
 ```
```
Comment (BenGitsCode) [inline diagnostic.md:26]: no need for header here.

File: diagnostic.md [hunk 2]
```diff
@@ -36,13 +42,19 @@ Now, we want to get a **single donut** from the server. Write the `curl` request
 you'd use to retrieve a **single donut**, using whatever ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request GET http://www.example.com/donuts/4 \
+--header "Content-Type: application/json"
 ```
```
Comment (BenGitsCode) [inline diagnostic.md:47]: same with header.

File: diagnostic.md [hunk 3]
```diff
@@ -51,13 +63,19 @@ Write the `curl` request you'd use to delete a single donut, using whatever
 ID you'd like.
 
 ```sh
-# your answer here
+curl --include --request DELETE "http://www.example.com/donuts/4" \
+--header "Content-Type: application/json"
```
Comment (BenGitsCode) [inline diagnostic.md:67]: still no header needed.

File: diagnostic.md [hunk 4]
```diff
@@ -73,14 +91,27 @@ data in JSON format.
...
 ```js
-let createDonut = /* your answer here */;
+let createDonut = function(data){
+  return $.ajax({
+    url: http://www.example.com + '/donuts/',
+    method: 'POST',
+    data,
+  });
+};;
 ```
```
Comment (BenGitsCode) [inline diagnostic.md:115]: Data is not defined.

```js
let createDonut = function () {
  return $.ajax({
    url: 'http://example.com/donuts',
    method: 'POST',
    data: {
      "donut" {
        "name": "French Cruller",
        "price": "$0.99"
      }
    }
  });
};

```

File: diagnostic.md [hunk 5]
```diff
@@ -100,11 +131,23 @@ object.
...
 ```js
-let changeDonut = /* your answer here */;
+let changeDonut = function(id, data){
+  return $.ajax({
+    url: http://www.example.com + '/donuts/' + id,
+    method: 'PATCH',
+    data,
+  });
+};;
 ```
```
Comment (BenGitsCode) [inline diagnostic.md:153]: data is not defined.

```js
let changeDonut = function (donutID) {
  return $.ajax({
    url: 'http://example.com/donuts/' + donutID,
    method: 'PATCH',
    data: {
      "donut" {
        "name": "Krüller",
      }
    }
  });
};
```

**Review by BenGitsCode — COMMENTED** [review]
7/10 great job! 👍

**Comment by doremaxime** [comment]
Ben,

I thought the instructions were to include those headers no? "include HTTP
headers in its output."

Max
WDI 16

**Comment by BenGitsCode** [comment]
You're right, our solutions don't include any but the instructions do say to include them. I'll revise the score with that in mind. Thanks for the feedback!

**Pull request closed without merging**